### ISSUE TITLE
refetchEvents triggers scroll reset in list view

### ISSUE THREAD

**Issue by marianabm** [fullcalendar/fullcalendar#4475]
Hi,
I'm using FullCalendar list view with contentHeight property.
 After call .fullCalendar( 'refetchEvents' ) scroll the window to top.

To reproduce: https://jsfiddle.net/marianabm/956e1uyh/

Thanks.

**Comment by acerix** [comment]
If you are looking for the list view to keep the same scroll position after the events change, you could store the scroll position and restore it after the events are rendered.  I think it makes sense to scroll to the top by default since the list of events could be completely different.

**Comment by marianabm** [comment]
I'm already doing that. I thought this wasn't the right behavior based on the other views behavior.
But you have a point, I hadn't thought the list could be completely different.

**Comment by daanbouws1** [comment]
I'm having a similar issue. We have recently upgraded from v3 to v4 and we used to solve the srcolling up problem in the eventAfterAllRendered callback, which was removed in v4. I've tried moving the $().scrollTop() call to the eventPositioned callback, but this has a significant delay before scrolling back down.
Our list view is hooked up to a socket that typically fires multiple times a minute causing a refetchEvents. This means our component has a fairly poor user experience as of right now as the scrollbar is scrolling all over the place. 
Did anyone find a more sensible solution for this? or does anyone @fullcalendar have a tip on how to solve this?
